app/views/help.mustache

```diff
@@ -29,6 +29,7 @@
       author or the uploader has the specified e-mail address.</li>
   </ul>
   <h2 id="scoring">Scoring</h2>
+  <p><i>NOTE: The Pub scoring model is still being fine-tuned, and is subject to change.</i></p>
```

Comment:
This does suggest a final scoring model might happen, which probably isn't true.

Reply:
s/still being fine-tuned/is under development/ ?